doc-content/jbpm-docs/src/main/asciidoc/BPMN2/proc-test-bus-process.adoc

```diff
@@ -0,0 +1,201 @@
+    }
+
+    @Test
+    public void testProcess() {
+
+        createRuntimeManager("hello.bpmn");
+
+        RuntimeEngine runtimeEngine = getRuntimeEngine();
+
+        KieSession ksession = runtimeEngine.getKieSession();
+
+        ProcessInstance processInstance = ksession.startProcess("com.sample.bpmn.hello");
+
+        assertProcessInstanceCompleted(processInstance.getId(), ksession);
+
+        assertNodeTriggered(processInstance.getId(), "StartProcess", "Hello", "EndProcess");
+    }
+}
+----
+
+`JbpmJUnitBaseTestCase` supports all predefined `RuntimeManager` strategies as part of the unit testing. Therefore, it is enough to specify the strategy that is used when you create a `RuntimeManager` as part of a single test.
+The following example shows the use of the PerProcessInstance strategy in a task service to manage user tasks:
+
+[source,java]
+----
+public class ProcessHumanTaskTest extends JbpmJUnitBaseTestCase {
+
+    private static final Logger logger = LoggerFactory.getLogger(ProcessHumanTaskTest.class);
+
+    public ProcessHumanTaskTest() {
+        super(true, false);
+    }
+
+    @Test
+    public void testProcessProcessInstanceStrategy() {
+        RuntimeManager manager = createRuntimeManager(Strategy.PROCESS_INSTANCE, "manager", "humantask.bpmn");
+        RuntimeEngine runtimeEngine = getRuntimeEngine(ProcessInstanceIdContext.get());
+        KieSession ksession = runtimeEngine.getKieSession();
+        TaskService taskService = runtimeEngine.getTaskService();
+
+        int ksessionID = ksession.getId();
+        ProcessInstance processInstance = ksession.startProcess("com.sample.bpmn.hello");
+
+        assertProcessInstanceActive(processInstance.getId(), ksession);
+        assertNodeTriggered(processInstance.getId(), "Start", "Task 1");
+
+        manager.disposeRuntimeEngine(runtimeEngine);
+        runtimeEngine = getRuntimeEngine(ProcessInstanceIdContext.get(processInstance.getId()));
+
+        ksession = runtimeEngine.getKieSession();
+        taskService = runtimeEngine.getTaskService();
+
+        assertEquals(ksessionID, ksession.getId());
+
+        // let John execute Task 1
+        List<TaskSummary> list = taskService.getTasksAssignedAsPotentialOwner("john", "en-UK");
+        TaskSummary task = list.get(0);
+        logger.info("John is executing task {}", task.getName());
+        taskService.start(task.getId(), "john");
+        taskService.complete(task.getId(), "john", null);
+
+        assertNodeTriggered(processInstance.getId(), "Task 2");
+
+        // let Mary execute Task 2
+        list = taskService.getTasksAssignedAsPotentialOwner("mary", "en-UK");
+        task = list.get(0);
+        logger.info("Mary is executing task {}", task.getName());
+        taskService.start(task.getId(), "mary");
+        taskService.complete(task.getId(), "mary", null);
+
+        assertNodeTriggered(processInstance.getId(), "End");
+        assertProcessInstanceCompleted(processInstance.getId(), ksession);
```

Comment:
Change this to `assertProcessInstanceNotActive(processInstance.getId(), ksession);` please. The current method is [deprecated](https://github.com/kiegroup/jbpm/blob/06f1719c5fcaa8e0c6bb8f464a5dc4f8143e8bc4/jbpm-test/src/main/java/org/jbpm/test/JbpmJUnitBaseTestCase.java#L601-L609).

Reply:
Done

Reply:
This is still here.